Documentation/teaching/labs/device_model.rst

```diff
@@ -90,7 +90,37 @@ are as follows:
 
 Other attributes are available, depending on the bus and driver used.
 
-.. image:: plug_and_play-sysfs.png
+.. ditaa::
+                                                                   +------+
+                                                                   | /sys |
+                                                                   +--+---+
+                                                                      |
+                 +----------------------------------------------------+-------------------------------------+-----------------------------------------+
+                 |                                                    |                                     |                                         |
+                 v                                                    v                                     v                                         v
+              +-----+                                             +-------+                            +---------+                                +--------+ 
+              | bus |                                             | class |                            | de|ices |                                | module |
+              +--+--+                                             +---+---+                            +----+----+                                +---+----+
+                 |                                                    |                                     |                                         |
+                 |                                                    |                                     |                       +-----------------+-----------------+
+                 |                                                    |                                     |                       |                                   |
+                 v                                                    v                                     v                       v                                   v
+   +--------------------------------+                      +-----------------------+            +-----------------------+     +----------------------+    +-------------------------+
+   | mydriver: struct device_driver |                      | myclass: struct class |            | mybus0: struct device |     | mybus: struct module |    | mydriver: struct module |
```

Comment:
In the old image, the first item was `mybus: struct bus_type`

Reply:
fixed